boards/arm/nrf52840_papyr/nrf52840_papyr_defconfig

```diff
@@ -0,0 +1,20 @@
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_NRF=y
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_NRF52840_PAPYR=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+CONFIG_UART_NRFX=y
```

Comment:
This is redundant. This option is enabled by default in `drivers/serial/Kconfig.nrfx`.